src/app/pages/office/cat-b/office.cat-b.page.html

```diff
@@ -1,11 +1,169 @@
 <ion-header>
-    <ion-toolbar>
-        <ion-title>office.cat-b</ion-title>
-    </ion-toolbar>
+  <practice-mode-banner *ngIf="isPracticeMode"></practice-mode-banner>
+  <ion-toolbar>
+    <ion-title>Office</ion-title>
+  </ion-toolbar>
 </ion-header>
 
-<ion-content>
-    <ion-button (click)="navigateBack()">Back</ion-button>
-    <ion-button (click)="navigateForward()">To Rekey Reason</ion-button>
-    <ion-button (click)="navigateJournal()">To Journal</ion-button>
+<ion-content forceOverscroll="false">
+
+  <office-candidate-section
+      [candidateName]="pageState.candidateName$ | async"
+      [driverNumber]="pageState.candidateDriverNumber$ | async"
+      [startTime]="pageState.startTime$ | async"
+      [testOutcomeText]="pageState.testOutcomeText$ | async">
+  </office-candidate-section>
+
+  <form [formGroup]="form">
+
+    <ion-card>
+      <ion-card-header>
+        <h4>Test summary</h4>
+      </ion-card-header>
+
+      <ion-card-content class="card-content-ios" no-padding>
```

Comment:
no-padding in Ionic 5 should go withing the class string, i.e.

class="card-content-ios ion-no-padding"